universal-application-tool-0.0.1/app/auth/FakeAdminClient.java

```diff
@@ -0,0 +1,38 @@
+package auth;
+
+import controllers.admin.routes;
+import java.util.Optional;
+import javax.inject.Inject;
+import org.pac4j.core.client.IndirectClient;
+import org.pac4j.core.credentials.AnonymousCredentials;
+import org.pac4j.core.util.HttpActionHelper;
+
+public class FakeAdminClient extends IndirectClient {
+
+  private ProfileFactory profileFactory;
+
+  @Inject
+  public FakeAdminClient(ProfileFactory profileFactory) {
+    this.profileFactory = profileFactory;
```

Comment:
idk how to suggest a change, but `checkNotNull` this constructor.

Reply:
I just learned this - it's one of the buttons next to all of the other comment formatting buttons:

![Screen Shot 2021-02-23 at 5 24 00 PM](https://user-images.githubusercontent.com/5732310/108931091-ebe7a500-75fb-11eb-898b-7a84b5e5bd21.png)

Reply:
Done!